behavior/MouseTracking/get_mouse_XY_pos.m

```diff
@@ -0,0 +1,75 @@
+function [ centroids ] = get_mouse_XY_pos( movie )
+% Extract the X,Y positions ( centroids ) of the mouse from the behavior
+%   video ( movie )
+%   
+% Input:
+%   movie: Behavior video (m4v); movie should be cropped (wavy curtain
+%   leads to fall centroids (haven't found another way to fix this)
+%
+% Returns matrix centroids where each row is an (x,y) coordinate of the
+%   mouse. 1st column = X, 2nd column = Y. Length of matrix =
+%   number of frames in movie
+%
+% Notes: Analyzes movie in chunks of 500 frames (smaller memory load)
+%   *** read function does not work properly on Linux, so unfortunately
+%   have to run this on a Windows machine ***
+%   *** Not very fast, but only have to run once; ~54K frames = ~40 min
+%
+% 2015-02-26 Fori Wang
+
+    behavior_vid = VideoReader(movie);
+    num_frames = behavior_vid.NumberofFrames;
+
+    % initialize centroids
+    centroids = cell(num_frames,1);
+    
+    % setup chunks of frames to read in movie
+    frame_indices = make_frame_indices(num_frames,1000);
```

Comment:
Now that you've renamed `make_frame_indices` into `make_frame_chunks`, you need to update these functions as well. While you are at it, can you replace `frame_indices` with `frame_chunks`?